http/src/main/resources/init-resources/startup.sh

```diff
@@ -127,6 +132,26 @@ then
     if [ ! -z "$JUPYTER_DOCKER_IMAGE" ] ; then
         echo "Restarting Jupyter Container $GOOGLE_PROJECT / $CLUSTER_NAME..."
 
+        # Make sure when runtimes restarts, they'll get a new version of jupyter docker compose file
```

Comment:
I don't know why, but by default the docker compose file was not being pulled when restarting the runtime. That can be problematic when users are trying to update their machine with different memory and CPU allocations.